README.md

```diff
@@ -179,6 +164,43 @@ MKdocs will generate static HTML files for Chainsafe markdown files located in `
 
 `make mkdocs`: Run MkDoc's docker image, building and hosting the html files on `localhost:8000`  
 
+# Local run
+## Prepare:
+
+1. Import keys:
```

Comment:
Add that app will connect to Devnet.

Reply:
Already fixed it. This is an old version of the file.